<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseType</class>
 <widget class="QDialog" name="ChooseType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>167</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Choose types to filter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBakery">
          <property name="text">
           <string>Bakery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkPetite">
          <property name="text">
           <string>Petite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkSalad">
          <property name="text">
           <string>Salad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBistroBox">
          <property name="text">
           <string>Bistro Box</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBreakfast">
          <property name="text">
           <string>Hot Breakfast</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkSandw">
          <property name="text">
           <string>Sandwich</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkPerfait">
          <property name="text">
           <string>Perfait</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chooseType">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
